### PR TITLE
docs: update changelog

### PR DESCRIPTION
**Pull request by dl239**
add v0.4.4 changelog
#1530

### CODE REVIEW

**Comment by lumianph** [comment]
image

add the 044 comparison at the end of CHANGELOG

**Review by lumianph — CHANGES_REQUESTED** [review]
add the 044 comparison at the end of CHANGELOG

**Review by tobegit3hub — APPROVED** [review]
LGTM

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,24 @@
 # Changelog
 
+## [0.4.4] - 2022-04-01
+
+### Features
+- Support the standalone version by Java and Python SDKs (#1302 #1325 #1485 @tobegit3hub @HuilinWu2 @keyu813)
+- Support the blocking execution for offline queries (#1486 @vagetablechicken )
+- Add the `getStatement` API in Java SDK (#1231 @dl239 )
+- Add the `executemany` API in Python SDK (#1402 @hezhaozhao-git )
```
Comment (lumianph) [inline CHANGELOG.md]: change this item to:
- Support multiple rows insertion in the Python SDK (#1402 @hezhaozhao-git )

**Comment by dl239** [comment]
> add the 044 comparison at the end of CHANGELOG

done

**Review by lumianph — APPROVED** [review]

**Pull request merged**